src/libs/antares/CMakeLists.txt

```diff
@@ -195,22 +195,43 @@ set(SRC_STUDY_AREA_UI
 source_group("study\\area\\ui" FILES ${SRC_STUDY_AREA_UI})
 
 
+set(SRC_STUDY_PART_COMMON
+    study/parts/common/series.h
+	study/parts/common/series.hxx
+	study/parts/common/series.cpp
+    study/parts/common/cluster.cpp
+    study/parts/common/cluster.h
+    study/parts/common/cluster_list.h

```

Comment:
There is no **cluster_list.hxx** in **SRC_STUDY_PART_COMMON** ?

Reply:
no